### PR TITLE
Release 0.25.0

### PR DESCRIPTION
**Pull request by ishustava**
* Update changelog
* Bump the consul-k8s image
* Clarify that the controller works with consul 1.8.4

### CODE REVIEW

**Review by thisisnotashwin — APPROVED** [review]
most excellent!! 🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ## Unreleased
 
+## 0.25.0 (Oct 12, 2020)
```
Comment (ashwin-venkatesh) [inline CHANGELOG.md:3]: I thought this got done by the CI automatically?

**Review by kschoche — APPROVED** [review]
Excellent work!
🍹

File: CHANGELOG.md (hunk 1 above)
Comment (ishustava) [reply CHANGELOG.md:3]: only in the consul-k8s pipeline!

Comment (ashwin-venkatesh) [reply CHANGELOG.md:3]: image

cool

Comment (kschoche) [reply CHANGELOG.md:3]: on consul-k8s this is the case, but it is manual for consul-helm

**Review by thisisnotashwin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ishustava) [reply CHANGELOG.md:3]: > (that did not show up the way i hoped i would lol)

lol it did in my email though!

**Pull request merged**